### PR TITLE
Support limited, pre-defined workflows.

### LINKED ISSUE #80: ability to process image with multiple models in sequence
We need to have the ability to, for instance, segment and image and then apply tracking, all in one pipeline.

### PR DESCRIPTION
**Pull request by willgraf**
Bump all `consumers`, `frontend`, `janitor`, and `autoscaler` to version `0.3.0`.  The frontend provides only options for "Segmentation" and "Tracking", while the consumers will determine the scale and label of the data before segmenting it and returning the zipped data.

The Janitor and Autoscaler have been updated to support multiple queues for easy extensibility with new consumers.

Added all possible environment variables to charts, wrapping in quotes as `False` causes unexpected behavior.

Moved `data-processing` helmfile to `0399` to obviate that it is a deprecated service, and to get it out of the way of future consumers.

---

Fixes #80

### CODE REVIEW

**Comment by willgraf** [comment]
A few final TODOs:

- [ ] Update `tf-serving` to a stable release (currently on base image of tf-1.14).
- [ ] Update models to those trained with above version or lower.

File: conf/helmfile.d/0310.tf-serving.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ releases:
 
       image:
         repository: "vanvalenlab/kiosk-tf-serving"
-        tag: "0.1"
+        tag: "0.2.1"
```
Comment (dylanbannon) [inline conf/helmfile.d/0310.tf-serving.yaml]: Can we bump this to 0.3.0?

File: conf/helmfile.d/0250.tracking-consumer.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ releases:
 
       image:
         repository: "vanvalenlab/kiosk-redis-consumer"
-        tag: "0.2.2"
+        tag: "0.4.1"
```
Comment (dylanbannon) [inline conf/helmfile.d/0250.tracking-consumer.yaml]: Can we decrement this to 0.3.0?

File: conf/helmfile.d/0240.zip-consumer.yaml [hunk 3]
```diff
@@ -30,7 +30,7 @@ releases:
 
       image:
         repository: "vanvalenlab/kiosk-redis-consumer"
-        tag: "0.2.2"
+        tag: "0.4.1"
```
Comment (dylanbannon) [inline conf/helmfile.d/0240.zip-consumer.yaml]: Can we decrement this to 0.3.0?

File: conf/helmfile.d/0230.redis-consumer.yaml [hunk 4]
```diff
@@ -30,7 +30,7 @@ releases:
 
       image:
         repository: "vanvalenlab/kiosk-redis-consumer"
-        tag: "0.2.2"
+        tag: "0.4.1"
```
Comment (dylanbannon) [inline conf/helmfile.d/0230.redis-consumer.yaml]: Can we decrement this to 0.3.0?

File: conf/charts/redis-janitor/values.yaml [hunk 5]
```diff
@@ -24,6 +24,7 @@ rbac:
   pspEnabled: false
 
 env:
+  QUEUES: "predict,track"
```
Comment (dylanbannon) [inline conf/charts/redis-janitor/values.yaml]: Do we need to have a "train" queue listed here, since the autoscaler also has a "train" queue?

Comment (willgraf) [reply conf/charts/redis-janitor/values.yaml]: The "train" flow has been disabled for now, but we could potentially have it check the key anyway (there just will be no values).

**Comment by willgraf** [comment]
In regards to the release tags, they do indicate specific versions of the software, so changing them to 0.3.0 will not be using the desired software versions.

`kiosk-tf-serving` could be updated to 0.3.0, though there really is no code change to justify a new release.

There have been a few changes for the `redis-consumers` that have led to new releases.  Today, `0.4.1` is the latest, but we will likely have a bugfix in tracking that will lead to a `0.4.2` release. (In light of this, maybe we should hold off on this PR for a couple of days.

**Pull request merged**